index/static/petition/style.css

```diff
@@ -91,3 +91,14 @@ body {
 .card {
     min-height: 175px;
 }
+
+.headingbar{
+    text-align: center;
+    /**box-shadow: 0px 2.5px rgba(255, 0, 0, 0.212);**/
+}
+
+/**about.html-only tags**/
+.rule{
```

Comment:
I would remove this class. See comment in `index/templates/about.html` on line 32.